app/views/groups/samples/index.html.erb

```diff
@@ -172,7 +172,7 @@
                   class: "max-sm:grow",
                 ) do |f| %>
           <input type="hidden" name="format" value="turbo_stream"/>
-          <%= f.submit t(".deselect_all_button"),
+          <%= f.submit t('common.controls.deselect_all'),
                    class: "button button-default w-full",
                    title: t(".deselect_all_tooltip") %>
```

Comment:
Suggestion (blocking): The title for deselect all button is the same as the button text (which does not match project samples page where title is more descriptive).

Reply:
Updated in 873ae66bb3865a956bc9520d07c394f1354ca1db